client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -17,14 +17,17 @@ import {
  * Internal dependencies
  */
 import { getContactDetailsExtraCache } from 'state/selectors';
+import { getCurrentUserLocale } from 'state/current-user/selectors';
 import { updateContactDetailsCache } from 'state/domains/management/actions';
 import FormFieldset from 'components/forms/form-fieldset';
 import FormLabel from 'components/forms/form-label';
 import FormSelect from 'components/forms/form-select';
 import FormCheckbox from 'components/forms/form-checkbox';
+import FormInputValidation from 'components/forms/form-input-validation';
 
 const ciraAgreementUrl = 'https://services.cira.ca/agree/agreement/agreementVersion2.0.jsp';
 const defaultValues = {
+	lang: 'EN',
 	legalType: 'CCT',
 	ciraAgreementAccepted: false,
```

Comment:
Oh, that answers my question [here](https://github.com/Automattic/wp-calypso/pull/15412#discussion_r123990171). Yeah, let's make the default true, it's ok
(on wp.com terms of service, there's not even a checkbox)